src/library.py

```diff
@@ -833,30 +833,22 @@ def day_plot_obo(self, plt_pts):
         pass
 
 
-class DemandControlVentilation(CheckLibBase):
-    points = [
-        "v_oa",
-        "s_ahu",
-        "s_eco",
-        "no_of_occ_per1",
-        "no_of_occ_per2",
-        "no_of_occ_per3",
-        "no_of_occ_per4",
-        "no_of_occ_core",
-    ]
-
+class DemandControlVentilation(RuleCheckBase):
     def verify(self):
         df_filtered = self.df.loc[
             (self.df["s_eco"] == 0.0) & (self.df["s_ahu"] != 0.0)
         ]  # filter out data when economizer isn't enabled
 
-        df_filtered["no_of_occ"] = (
-            df_filtered["no_of_occ_per1"]
-            + df_filtered["no_of_occ_per2"]
-            + df_filtered["no_of_occ_per3"]
-            + df_filtered["no_of_occ_per4"]
-            + df_filtered["no_of_occ_core"]
-        )
+        df_filtered["no_of_occ"] = 0.0
```

Comment:
Where does this 'no_of_occ' come from?

Reply:
It's a summation of all the number of occupants. I found this naming is confusing so, I changed ```no_of_occ``` to ```sum_no_of_occ```.